Task2/aga/TicTacToe/TicTacToe/TheGame.xaml.cs

```diff
@@ -0,0 +1,242 @@
+                click.TurnCounter();
+                displayTurn(turn);
+            }
+            else
+                { displayMessageBox(param, turn); }
+            
+        }
+
+        private bool wasClicked(int param)
+        {
+            return clickedButtons[param];           
+        }
+
+        private bool wasWinned()
+        {
+            if (A1.Content == A2.Content && A1.Content == A3.Content && A1.Content!=null)
+                {
+                    winner = A1.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (B1.Content == B2.Content && B1.Content == B3.Content && B1.Content != null)
+                {
+                    winner = B1.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (C1.Content == C2.Content && C1.Content == C3.Content && C1.Content != null)
+                {
+                    winner = C1.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (A1.Content == B1.Content && A1.Content == C1.Content && A1.Content != null)
+                {
+                    winner = A1.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (A2.Content == B2.Content && A2.Content == C2.Content && A2.Content != null)
+                {
+                    winner = A2.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (A3.Content == B3.Content && A3.Content == C3.Content && A3.Content != null)
+                {
+                    winner = A3.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (A1.Content == B2.Content && A1.Content == C3.Content && A1.Content != null)
+                {
+                    winner = A1.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else if (A3.Content == B2.Content && A3.Content == C1.Content && A3.Content != null)
+                {
+                    winner = A3.Content;
+                    tryAgain.Visibility = Visibility.Visible;
+                    return true;
+                }
+            else
+                { return false; }
+        }
+
+        private void tryAgain_Click(object sender, RoutedEventArgs e)
+        {
+            click = new Click();
+            messageBox.Content = "Player X turn";
+            diplayTurn.Content = "";
+            A1.Content = B1.Content = C1.Content = A2.Content = B2.Content = C2.Content = A3.Content = B3.Content = C3.Content = null;
+            for (int i = 0; i<9;i++)
```

Comment:
while it's obvious what's the number 9 representing here, try to keep it even more obvious by not using magic numbers, but keeping the value in a properly named variables/fields.